theano/sandbox/cuda/opt.py

```diff
@@ -1128,11 +1128,9 @@ def local_gpu_advanced_incsubtensor1(node):
             compute_capability = device_properties(active_device_no)['major']
             if (compute_capability < 2 or y.ndim != 2 or x.ndim != 2):
 
-                gpu_op = GpuAdvancedIncSubtensor1(
-                    set_instead_of_inc=set_instead_of_inc)
+                gpu_op = tensor.AdvancedIncSubtensor1(**node.op._props_dict())
```

Comment:
Don't use the CPU op and use props. The parent class AdvancedIncSubtensor1 have \__props__ and init. I don't know where you got the information you wrote. I double checked and I have the inverse information.